src/utils/toast.js

```diff
@@ -0,0 +1,33 @@
+export const openToast = ({ message, type = "success" }) => {
+  const successToast = document.querySelector("#success-toast");
+  const errorToast = document.querySelector("#error-toast");
+  const infoToast = document.querySelector("#info-toast");
+  const toastMessage = document.querySelector("#toast-message");
+
+  toastMessage.textContent = message;
+  if (type === "success") {
+    successToast.classList.remove("opacity-0");
+    successToast.classList.add("opacity-100");
+
+    setTimeout(() => {
+      successToast.classList.remove("opacity-100");
+      successToast.classList.add("opacity-0");
+    }, 3000);
+  } else if (type === "error") {
+    errorToast.classList.remove("opacity-0");
+    errorToast.classList.add("opacity-100");
+
+    setTimeout(() => {
+      errorToast.classList.remove("opacity-100");
+      errorToast.classList.add("opacity-0");
+    }, 3000);
+  } else if (type === "info") {
+    infoToast.classList.remove("opacity-0");
+    infoToast.classList.add("opacity-100");
+
+    setTimeout(() => {
+      infoToast.classList.remove("opacity-100");
+      infoToast.classList.add("opacity-0");
+    }, 3000);
+  }
+};
```

Comment:
중복된 내용이 많습니다. 아래처럼 개선해볼 수도 있을 것 같아요.

```suggestion
export const openToast = ({ message, type = "success" }) => {
  const successToast = document.querySelector("#success-toast");
  const errorToast = document.querySelector("#error-toast");
  const infoToast = document.querySelector("#info-toast");
  const toastMessage = document.querySelector("#toast-message");
  let targetToast;

  toastMessage.textContent = message;
  if (type === "success") {
    targetToast = successTost;
  } else if (type === "error") {
    targetToast = errorToast;
  } else if (type === "info") {
    targetToast = infoToast;
  }
  
  targetToast.classList.remove("opacity-0");
  targetToast.classList.add("opacity-100");
  setTimeout(() => {
    targetToast.classList.remove("opacity-100");
    targetToast.classList.add("opacity-0");
  }, 3000);
};
```